packages/web/src/pages/sign-up-page/pages/CreatePasswordPage.tsx

```diff
@@ -223,142 +233,156 @@ export const CreatePasswordPage = () => {
-                      onBlur={(e) => {
-                        formikHandleBlur(e)
-                        handleConfirmPasswordBlur(values)
-                      }}
-                      label={messages.confirmPasswordLabel}
-                      value={values.confirmPassword}
-                      error={touched.confirmPassword && hasConfirmPasswordError}
-                    />
-                  </Flex>
-                  <Flex mt='l' gap='l' direction='column'>
-                    {requirements.map((req) => (
-                      <CompletionChecklistItem
-                        key={req.key}
-                        status={req.status}
-                        label={req.label}
-                      />
-                    ))}
-                  </Flex>
-                </Box>
+                    <Box>
+                      <Flex
+                        direction='column'
+                        gap='l'
+                        className={styles.inputsContainer}
+                      >
+                        <TextInput
+                          name='password'
+                          type='password'
+                          autoFocus
+                          autoComplete='new-password'
+                          onChange={(e) => {
+                            setFieldValue('password', e.target.value)
+                            handlePasswordChange({
+                              password: e.target.value,
+                              confirmPassword: values.confirmPassword
+                            })
+                          }}
+                          onBlur={(e) => {
+                            formikHandleBlur(e)
+                            handlePasswordBlur(values)
+                          }}
+                          label={messages.passwordLabel}
+                          value={values.password}
+                          error={touched.password && hasPasswordError}
+                        />
+                        <TextInput
+                          name='confirmPassword'
+                          type='password'
+                          autoComplete='new-password'
+                          onChange={(e) => {
+                            setFieldValue('confirmPassword', e.target.value)
+                            handleConfirmPasswordChange({
+                              password: values.password,
+                              confirmPassword: e.target.value
+                            })
+                          }}
+                          onBlur={(e) => {
+                            formikHandleBlur(e)
+                            handleConfirmPasswordBlur(values)
+                          }}
+                          label={messages.confirmPasswordLabel}
+                          value={values.confirmPassword}
+                          error={
+                            touched.confirmPassword && hasConfirmPasswordError
+                          }
```

Comment:
Could we factor out the password field into its own component?
This jsx is getting long and this seems like the best candidate for extraction

Reply:
Yeah and maybe not for this pr since it's about the wrapper, but a good item to add for follow on pr

Reply:
sg!